src/ConfigurationContext.tsx

```diff
@@ -61,6 +64,7 @@ const ConfigurationProvider: FunctionComponent = ({ children }) => {
         displayReportAnIssue,
         displaySelfAssessment,
         healthAuthorityAdviceUrl,
+        healthAuthorityLearnMoreUrl: learnMoreUrl || healthAuthorityAdviceUrl,
```

Comment:
This jumps out as a bit subtle and prone to error/confusion.  Should we default to the advice url? or would it be better to enforce providing a url and otherwise not see a url at all? We could always duplicate the url in the config file itself if we want both to be the same.

Reply:
@johnschoeman @aledustet Would you all recommend that I update the mobile resources repo to set the LEARN_MORE _URL to the AUTHORITY_ADVICE_URL in order to keep the current behavior for other HAs?

Reply:
Yes, i think that makes sense to do.